app/validators/claim/transfer_claim_sub_model_validator.rb

```diff
@@ -5,11 +5,7 @@ def has_one_association_names_for_steps
       case_details: [],
       defendants: [],
       offence_details: [],
-      fees: %i[
-        transfer_fee
-        assessment
```

Comment:
[Ditto](https://github.com/ministryofjustice/Claim-for-Crown-Court-Defence/pull/2252/files#r170615857)